app/src/main/kotlin/com/wire/android/ui/common/bottomsheet/conversation/HomeSheetContent.kt

```diff
@@ -108,21 +110,55 @@ internal fun ConversationMainSheetContent(
                     )
                 }
             }
+
+            if (conversationSheetContent.canAddToFavourite() && !conversationSheetContent.isArchived) {
+                conversationSheetContent.isFavorite?.let { isFavorite ->
+                    if (isFavorite) {
```

Comment:
in fact there are only 2 differences in this `if else`: `title` of item and boolean parameter for `changeFavoriteState`. 
So it will save a lot of lines to have something like:
```
add {
    MenuBottomSheetItem(
        title = stringResource(if (isFavorite) R.string.label_remove_from_favourites else R.string.label_add_to_favourites),
        leading = {
            MenuItemIcon(
                id = R.drawable.ic_favourite,
                contentDescription = stringResource(R.string.content_description_remove_from_favourites),
            )
        },
        onItemClick = {
            changeFavoriteState(
                GroupDialogState(
                    conversationSheetContent.conversationId,
                    conversationSheetContent.title
                ),
                !isFavorite
            )
        }
    )
}      
```